spec/controllers/account/tasks_controller_spec.rb

```diff
@@ -58,6 +58,20 @@
       expect(task1.creator).to eq(user1)
     end
 
+    it 'should create section' do
+      post :create, params: {
+        project_id: project.id,
+        task: {
+          title: Faker::Lorem.sentence,
+          description: Faker::Lorem.paragraph,
+          section: true
+        }
+      }
+      expect{ post :create, params: { project_id: project.id, task: { title: Faker::Lorem.sentence,
+        description: Faker::Lorem.paragraph, section: true }}}.to change(Task, :count).by(1)
```

Comment:
still not good..
why wouldn't u try `change { Task.all.section.count }.by(1)` or `change { Task.all.where(section: true).count }.by(1)`